src/gsy_e/models/strategy/state/storage_state.py

```diff
@@ -453,3 +453,19 @@ def get_results_dict(self, current_time_slot: DateTime) -> dict:
         return {
             "soc_history_%": self.charge_history.get(current_time_slot, 0)
         }
+
+
+class ScmStorageState(StateInterface):
```

Comment:
Out of curiosity, is the empty class needed in this case ? Is it just for having a state object, thus allowing the EndpointBuffer class to access all states the same way?

Reply:
Exactly, I am not proud of this. but it is needed unfortunately.

Reply:
In this case though, I think that it can either be better named by this being called "DummyState" , since it does not do anything. Or possibly using default implementation on these abstractmethods.

Reply:
> Or possibly using default implementation on these abstractmethods.

I tried using `StateInterface` directly, but I remember It was not allowed as it was an ABC. I try again, if not, I will rename indeed.

Reply:
Might be because the abstractmethods do not have a default implementation, regardless though I think it is better to create a dummy class that essentially disables the interface. Thanks!